### ISSUE TITLE
Identically-named questions from differently-named repeating groups are merged

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/formpack#206]
Support request: https://community.kobotoolbox.org/t/download-export-error/5931/6
Internal discussion: https://www.flowdock.com/app/kobotoolbox/support/threads/sV6RLGs18xEw-FAbY_RJ4x3CMkB
In the wild: https://sentry.kbtdev.org/kobo/kpi-backend/issues/304938/

Minimal way to reproduce:
```python
import formpack
v1 = {'version': 'v1', 'content': {'survey': [
  {'name': 'repeat', 'type': 'begin_repeat'},
  {'name': 'hello', 'type': 'text', 'label': 'hello'},
  {'type': 'end_repeat'},
]}}
v2 = {'version': 'v2', 'content': {'survey': [
  {'name': 'repeat_oops', 'type': 'begin_repeat'},
  {'name': 'hello', 'type': 'text', 'label': 'hello'},
  {'type': 'end_repeat'},
]}}
fp = formpack.FormPack(versions=[v1, v2])
x = fp.export(versions=fp.versions.keys())
sub = {'__version__': 'v1', 'repeat': [
  {'repeat/hello': 'hi there'}, {'repeat/hello': 'how do you do'}
]}
x.to_xlsx('/dev/null', [sub]) # will fail
```
Demonstration:
```
(packenv) john@world:/tmp/formpack$ git show --oneline
7bd244c (HEAD -> master, origin/master, origin/HEAD) Merge pull request #201 from kobotoolbox/200-geojson-export

(packenv) john@world:/tmp/formpack$ ipython
Python 2.7.15+ (default, Jul  9 2019, 16:51:35) 
Type "copyright", "credits" or "license" for more information.

IPython 5.8.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import formpack
   ...: v1 = {'version': 'v1', 'content': {'survey': [
   ...:   {'name': 'repeat', 'type': 'begin_repeat'},
   ...:   {'name': 'hello', 'type': 'text', 'label': 'hello'},
   ...:   {'type': 'end_repeat'},
   ...: ]}}
   ...: v2 = {'version': 'v2', 'content': {'survey': [
   ...:   {'name': 'repeat_oops', 'type': 'begin_repeat'},
   ...:   {'name': 'hello', 'type': 'text', 'label': 'hello'},
   ...:   {'type': 'end_repeat'},
   ...: ]}}
   ...: fp = formpack.FormPack(versions=[v1, v2])
   ...: x = fp.export(versions=fp.versions.keys())
   ...: sub = {'__version__': 'v1', 'repeat': [
   ...:   {'repeat/hello': 'hi there'}, {'repeat/hello': 'how do you do'}
   ...: ]}
   ...: x.to_xlsx('/dev/null', [sub])
   ...: 
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-1-b4a55ac4a8b9> in <module>()
     15   {'repeat/hello': 'hi there'}, {'repeat/hello': 'how do you do'}
     16 ]}
---> 17 x.to_xlsx('/dev/null', [sub])

/tmp/formpack/src/formpack/reporting/export.pyc in to_xlsx(self, filename, submissions)
    625             sheet_row_positions[sheet_] = row_index
    626 
--> 627         for chunk in self.parse_submissions(submissions):
    628             for section_name, rows in chunk.items():
    629                 try:

/tmp/formpack/src/formpack/reporting/export.pyc in parse_submissions(self, submissions)
    140         self.reset()
    141         for submission in submissions:
--> 142             formatted_chunks = self.parse_one_submission(submission)
    143             if not formatted_chunks:
    144                 continue

/tmp/formpack/src/formpack/reporting/export.pyc in parse_one_submission(self, submission, version)
    131         section = version.sections.values()[0]
    132         submission = FormSubmission(submission)
--> 133         return self.format_one_submission([submission.data], section)
    134 
    135     def parse_submissions(self, submissions):

/tmp/formpack/src/formpack/reporting/export.pyc in format_one_submission(self, submission, current_section)
    309         _section_name = current_section.name
    310         _lang = self.lang
--> 311         _empty_row = self._empty_row[_section_name]
    312         _indexes = self._indexes
    313         row = self._row_cache[_section_name]

KeyError: u'Submissions'
```

**Comment by jnm** [comment]
maybe applies to regular groups as well? see https://www.flowdock.com/app/kobotoolbox/kobo/threads/DxrUvkoLiD3vPFhDX9isK0dqh9n

**Comment by joshuaberetta** [comment]
Closed by #276
